### PR TITLE
further entrypoint.sh templating improvements

### PR DESCRIPTION
**Pull request by gdams**
removes the JDK8 specific if statement

### CODE REVIEW

**Review by steelhead31 — APPROVED** [review]
Looks ok.

File: 8/jre/alpine/entrypoint.sh [hunk 1]
```diff
@@ -28,11 +28,6 @@ TMPDIR=${TMPDIR:-/tmp}
 # JDK truststore location
 JRE_CACERTS_PATH=$JAVA_HOME/lib/security/cacerts
 
-# JDK8 puts its JRE in a subdirectory
```
Comment (karianna) [inline 8/jre/alpine/entrypoint.sh:31]: This change doesn't seem to mirror what other platforms are doing? e.g. the ubuntu noble one above.

Comment (tellison) [reply 8/jre/alpine/entrypoint.sh:31]: The 'ubuntu noble one' is a JDK rather than a JRE

File: docker_templates/entrypoint.sh.j2 [hunk 2]
```diff
@@ -13,12 +13,12 @@ set -e
 TMPDIR=${TMPDIR:-/tmp}
 
 # JDK truststore location
```
Comment (tellison) [inline docker_templates/entrypoint.sh.j2:15]: Remove ambiguity in comment.

```suggestion
# Truststore location
```

**Review by tellison — APPROVED** [review]
Minor comment change suggestion.

**Comment by tellison** [comment]
Waiting for @karianna to concur on the approval.

**Pull request merged**